src/plugin/modules/widgets/kbaseCatalogStats.js

```diff
@@ -874,7 +874,7 @@ define([
 
                     // Creates a job log viewer button for any job which has been or is being
                     // run.
-                    if (job.exec_start_time) {
+                    if (job.exec_start_time || job.status === 'canceled by user' ) {
```

Comment:
are there other job cancelation conditions (e.g. canceled by an admin) which generate different status values? I just implemented this in a rewrite of the job browser and matched against the string starting with 'canceled' just in case .

Reply:
Actually this part is not necessary anymore since job.exec_start time will now exist for all jobs.  I can delete it.